cirq-core/cirq/value/probability.py

```diff
@@ -33,3 +35,7 @@ def validate_probability(p: float, p_str: str) -> float:
     elif p > 1:
         raise ValueError(f'{p_str} was greater than 1.')
     return p
+def state_vector_to_probabilities(state_vector: np.ndarray) -> np.ndarray:
```

Comment:
I think we should be using `cirq.to_valid_state_vector` to convert the argument (which should be of type `'cirq.STATE_VECTOR_LIKE'`)  to a valid state vector. 
After that there is no need for `validate_probability` either.

Reply:
Agree. This is being done on https://github.com/quantumlib/Cirq/pull/4439.